### DISCUSSION TITLE
Cookie _plausible_key

### DISCUSSION THREAD

**Discussion by michaelkamleitner** [plausible/analytics#1862] [Anything Else]
Hi, I'm just getting started testing the cloud-based Plausible.
Really like it so far! 💕

However I noticed it sets a cookie "_plausible_key" when embedded on my site. 
Can someone explain the purpose of this cookie, and whether it's possible to disable it?

Thanks in advance, and sorry of this is an obvious one!

**Comment by metmarkosaric** [comment]
hi @michaelkamleitner and glad you're enjoying Plausible! this cookie is the one we use on our site to keep people logged in so they don't need to manually log in every time. @ukutaht can perhaps share more on how this works with the embed dashboard

**Comment by michaelkamleitner** [comment]
Ahh my bad, that makes sense of course (I obviously forgot to test this in incognito mode).
perfect, thanks so much :)

**Reply by metmarkosaric** [reply]
no problem!
